### ISSUE TITLE
put_location does not respect URL

### ISSUE THREAD

**Issue by Egor3f** [notroj/litmus#23]
**Describe the bug**
Make basic test with URL specified, for example, to `http://localhost:9999/api/fileserver/webdav/`
Get an error on test 6: FAIL (PUT to create resource MUST return 201 [RFC9110ẞ9.3.4])
That's because put_location does not treat url and makes request to `/dav/put%23test`

**Environment**
 - OS: Docker image with bookworm
 - SSL library version: OpenSSL 3.0.11

**To Reproduce**
make URL=http://localhost:12345/some/path check

**Debugging output**
```
Running pre_send hooks
Sending request headers:
PUT /dav/put%23test HTTP/1.1
User-Agent: litmus/0.15 neon/0.33.0
Connection: TE
TE: trailers
Host: localhost:12345
Content-Length: 5
X-Litmus: basic: 6 (put_location)

Sending request-line and headers:
Sending request body:
Body block (5 bytes):
[hello]
Request sent; retry is 1.
[status-line] < HTTP/1.1 404 Not Found
```

**Comment by Egor3f** [comment]
there is already a pull request for this bug, please, approve it https://github.com/notroj/litmus/pull/20

**Comment by notroj** [comment]
Fixed in 601068979a9501e46d6ec3671c49216413a644cb with thanks to @Egor3f

**Comment by notroj** [comment]
With thanks to @vobruba-martin !
